control/planktoscopehat/planktoscope/camera/mqtt.py

```diff
@@ -89,14 +89,11 @@ def run(self) -> None:
         streaming_thread.start()
 
         loguru.logger.info("Starting the MQTT backend...")
-        # TODO(ethanjli): expose the camera settings over "camera/settings" instead! This requires
-        # removing the "settings" action from the "imager/image" route which is a breaking change
-        # to the MQTT API, so we'll do this later.
```

Comment:
Making it possible to change camera settings on the `camera/settings` topic rather than the `imager/image` topic is still an unfinished TODO item. Please revert your deletion of this code comment.